src/cr/cube/cubepart.py

```diff
@@ -900,6 +934,12 @@ def _columns_variance(self):
             self.counts / self.columns_margin * (1 - self.counts / self.columns_margin)
         )
 
+    @lazyproperty
+    def _row_variance(self):
+        """ndarray of variances for row percentages"""
+        margin = self.rows_margin[:, None]
```

Comment:
I don't understand why are you adding an extra dimension here? Can you explain please?
Furthermore I'd prefer to be more explicit here IMHO: `self._rows_margin[:, np.newaxis]` spells out better than using `None`. This way we can read better that we are adding an extra dimension

Reply:
Ah this property is not placed in its alphabetical right place. Please order it 😄

Reply:
Added comment